lacci/lib/shoes/app.rb

```diff
@@ -181,34 +185,33 @@ def current_draw_context
     # want to (and/or can't) take control of the event loop.
     def run
       if @do_shutdown
-        $stderr.puts "Destroy has already been signaled, but we just called Shoes::App.run!"
+        warn 'Destroy has already been signaled, but we just called Shoes::App.run!'
```

Comment:
Now I wonder if we ever shut down the logfile stuff. Because warn won't appear if logfile stuff isn't currently working. But this should only get called after the logs are initialised, so definitely no startup problem.